### ISSUE TITLE
Failure if photo face step is used, but microphone permission was denied

### ISSUE THREAD

**Issue by novemberborn** [onfido/onfido-ios-sdk#88]
### What was the expected behaviour?

A user has denied our application access to their microphone. They then enter an Onfido flow. We only use the photo face step, and they've authorized camera access, so they can complete the flow.

### What happened instead?

The flow fails because the lack of microphone permission.

### Version info:

- Cocoapods (if applicable):
- Onfido SDK (Debug/Release): 11.1.0
- iOS:
- Xcode:
- Device/Simulator:
- Device/Simulator language:

### Integration configuration:

```
OnfidoConfig.builder()
      .withToken("token")
      .withApplicantId("id")
      .withCustomLocalization()
      .withDocumentStep()
      .withFaceStep(ofVariant: .photo)
      .build()
```

### Steps to reproduce:

Please describe in as much detail as possible on how to reproduce the problem

* Outside of Onfido, prompt for microphone access
* Deny access
* Enter the Onfido flow

**Comment by anuragajwani** [comment]
Hi @novemberborn,

Thanks for raising the issue. We will investigate and get back to you.

Anurag

**Comment by anuragajwani** [comment]
Hi @novemberborn,

We have been able to reproduce the issue. We will update you when we have fixed this.

Sorry for the inconvenience,
Anurag

**Comment by novemberborn** [comment]
This is fixed in 11.1.2, thanks!
